.github/workflows/tests.yaml

```diff
@@ -94,7 +94,7 @@ jobs:
   # If you would like to make a change to the integration tests workflow, please do it in the era-compiler-ci repository
   integration-tests:
```

Comment:
We should also align the `target-machine` jobs in tester [here](https://github.com/matter-labs/era-compiler-tester/pull/87/files#diff-95557d53bf91069e59d82b9d8fcfaf52ec84a762858983edd5ef3c9b3e4c8191L74-L82) and in the llvm repo [here ](https://github.com/matter-labs/era-compiler-llvm/blob/main/.github/workflows/benchmarks-integration-tests.yml#L79-L103) to align with the update.

Please, let me know if you need help in that.